modules/rhdh-plugins-reference/assembly-rhdh-installing-dynamic-plugins.adoc

```diff
@@ -1,17 +1,16 @@
 [id="rhdh-installing-dynamic-plugins"]
-
 = Dynamic plugin installation
 
 The dynamic plugin support is based on the backend plugin manager package, which is a service that scans a configured root directory (`dynamicPlugins.rootDirectory` in the app config) for dynamic plugin packages and loads them dynamically.
 
-You can use the dynamic plugins that come preinstalled with {product} or install external dynamic plugins from a public NPM registry. 
+You can use the dynamic plugins that come preinstalled with {product} or install external dynamic plugins from a public NPM registry.
 
 //viewing installed plugins
 include::proc-viewing-installed-plugins.adoc[leveloffset=+1]
 //preinstalled plugins
 include::con-preinstalled-dynamic-plugins.adoc[leveloffset=+1]
-include::ref-rhdh-plugins.adoc[leveloffset=+2]
-//helm installed plugins 
+include::rhdh-supported-plugins.adoc[leveloffset=+2]
```

Comment:
I didn't really name this, I am just updating the filename to match what was already in the module ID because it was confusing to have the ID and filename be different (generally it's advised to keep them the same, per the [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings)).

If we're going to change this, we should change both, but I don't think it should be a blocker for this PR, since this is just a correction.